MineColab.ipynb

```diff
@@ -125,8 +125,9 @@
         "id": "Fy5-YjJMPV3S"
       },
       "source": [
-        "# Replace \"1.16.4\" with your desired server version.\n",
+        "# Replace \"1.17\" with your desired server version.\n",
         "# Available versions:\n",
+        "# - 1.17\n",
```

Comment:
Maybe adding "experimental builds, use with caution" would be desirable as paper's 1.17 servers are still in dev.
![image](https://user-images.githubusercontent.com/25097841/123091145-a078c280-d446-11eb-93b3-47d8266c67fd.png)